bootstrap/roles/appliance-build.bootstrap/tasks/main.yml

```diff
@@ -19,31 +19,94 @@
   register: toplevel
 
 - apt:
-    name: aptitude
+    name: software-properties-common
     state: present
+    update_cache: yes
 
-- apt:
+- apt_repository:
+    repo: ppa:ansible/ansible
     update_cache: yes
 
 - apt:
-    upgrade: yes
+    upgrade: dist
 
 - apt:
-    name: "{{ item }}"
+    name:
+      - ansible
+      - aptly
+      - awscli
+      - bc
+      - coreutils
+      - equivs
+      - gdisk
+      - git
+      - java-package
```

Comment:
We don't need java-package anymore as we don't build Java in appliance-build anymore.

Reply:
Probably. I'll remove, retest, and make sure things still work.